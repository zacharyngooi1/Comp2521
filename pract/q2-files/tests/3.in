50 80 70 90 25 10 40 
50 30 80 10 40 70 90 
50 30 10 40 80 90 70 


