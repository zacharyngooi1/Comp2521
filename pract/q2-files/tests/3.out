
Displaying t1 (sideways) 
		90
	80
		70
50
		40
	25
		10


Displaying t2 (sideways) 
		90
	80
		70
50
		40
	30
		10


Displaying t3 (sideways) 
		90
	80
		70
50
		40
	30
		10

 -------  
equalBST(t1, t2) returns: 0 
equalBST(t2, t3) returns: 1 
